icon-registry/icons/shape-heart_x16.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none"
+  xmlns="http://www.w3.org/2000/svg">
+  <path d="M4.44849 2C3.53603 2.00104 2.66124 2.36794 2.01603 3.0202L2.00916 3.01604C1.36301 3.66927 1 4.55523 1 5.47903C1 6.40283 1.36301 7.2888 2.00917 7.94202L7.99428 13.9926L7.99634 13.9905L8.00572 14L13.9908 7.94942C14.637 7.2962 15 6.41023 15 5.48643C15 4.56263 14.637 3.67667 13.9908 3.02344C13.671 2.6994 13.2912 2.44226 12.873 2.26673C12.4548 2.0912 12.0065 2.00072 11.5537 2.00046C11.1009 2.0002 10.6525 2.09017 10.2341 2.26522C9.81572 2.44028 9.43557 2.69698 9.1154 3.02066L8.00229 4.14595L6.88323 3.01465C6.23657 2.36386 5.36095 1.99896 4.44849 2Z" fill="#D0D2E0" stroke="currentStroke" class="icon-dark-stroke icon-light-fill" stroke-width="2" stroke-linejoin="round"/>
```

Comment:
```suggestion
  <path d="M4.44849 2C3.53603 2.00104 2.66124 2.36794 2.01603 3.0202L2.00916 3.01604C1.36301 3.66927 1 4.55523 1 5.47903C1 6.40283 1.36301 7.2888 2.00917 7.94202L7.99428 13.9926L7.99634 13.9905L8.00572 14L13.9908 7.94942C14.637 7.2962 15 6.41023 15 5.48643C15 4.56263 14.637 3.67667 13.9908 3.02344C13.671 2.6994 13.2912 2.44226 12.873 2.26673C12.4548 2.0912 12.0065 2.00072 11.5537 2.00046C11.1009 2.0002 10.6525 2.09017 10.2341 2.26522C9.81572 2.44028 9.43557 2.69698 9.1154 3.02066L8.00229 4.14595L6.88323 3.01465C6.23657 2.36386 5.36095 1.99896 4.44849 2Z" fill="#D0D2E0" stroke="currentColor" class="icon-dark-stroke icon-light-fill" stroke-width="2" stroke-linejoin="round"/>
```

Reply:
there are a number of these that say `currentStroke`. i assume they should all say `currentColor`?